### PR TITLE
adds committee roles

### PR DESCRIPTION
**Pull request by katyhuff**
just realized this was never made public... was supposed to be included in the call for steering committee nominations (that was the point).
can we add this here, @gvwilson , @DamienIrving ?

### CODE REVIEW

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 01:44:17PM -0800, Katy Huff wrote:
>   * adds committee roles

I think the chair's:

  4. Ensure that minutes are kept and distributed (by delegation or
     otherwise).

should be the secretary's responsibility.

**Comment by katyhuff** [comment]
It is already. This statement is just supposed to indicate that the chair is supposed to make sure that the responsibilities of others are being met. They don't have to actually do it themselves. The secretary is expected to do it. See point 1 under the secretary. This is what was reviewed by the interim steering committee last month and it was meant to match the chair responsibilities in that separate document already in this repo. It is not static. This document can be changed by the new steering committee as they deem fit.

**Comment by wking** [comment]
While I'm all for dedicated leaders, I don't see the point in getting
volunteers to sign a commitment message.  Why not just list these
criteria, and say they're implicitly agreed to?  For teeth, maybe a
vote of more than 50% of the committee or petition by more than 50% of
the voting SWC members would be sufficient to evict a dud committee
member and start a new election.  I'm agnostic about whether or not
that election should be for just the dud member or all committee
members.  Gentoo says all [1], but obviously the original committee
members could be re-elected during that election.

[1]: http://wiki.gentoo.org/wiki/GLEP:39#Specification

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 01:52:54PM -0800, Katy Huff wrote:
> It is already. This statement is just supposed to indicate that the
> chair is supposed to make sure that the responsibilities of others
> are being met.

Isn't that already covered by the chair's:

  6. Ensure that SCF Steering Committee members are aware of their
     governance duties and responsibilities.

Why have a separate entry to call out this particular issue?

> The secretary is expected to do it. See point 1 under the secretary.

That talks about “the formation of meeting minutes”, but the chair's
point 4 (which I think should be the secretary's responsibility) is
about archival and publishing (“kept and distributed”).

> This is what was reviewed by the interim steering committee last
> month and it was meant to match the chair responsibilities in that
> separate document already in this repo. It is not static. This
> document can be changed by the new steering committee as they deem
> fit.

Ah.  In that case, feel free to just let my review cook here until the
(new) committee wants to address it.  It's certainly not critical (and
I imagine they would have done fine without this document at all,
although I like having things spelled out where possible).

**Comment by katyhuff** [comment]
Hi @wking . I hear you.
This agreement was agreed upon by the interim steering committee because explicit dedication is superior to implicit dedication. It sounds like you believe the opposite. Could you elaborate on why you would rather it be implicit?

The ejection of dud members is currently left up to the discretion of the new committee. When they decide to implement a rule, they can certainly do so.

**Comment by katyhuff** [comment]
@wking 

concerning your worry about the secretary not being required to keep and distribute the meeting minutes in addition to making sure they are taken.... please see points 2 and 3 of the secretary's responsibilities. 

As far as letting your review cook, that sounds great, except I would like to make sure this gets pulled. Many people are (rightly) asking what the roles might look like on this committee. This document was supposed to be pulled into this repository before the nomination request announcement, but it was left behind in our private repository. To help people answer their question about what they will be responsible for, this needs to be in the repository asap, IMO. 

I think that keeping your comments in mind for the future committee is a great idea though, and I hope we can make sure, together, that they are made aware of your thoughts when the committee is formed.

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 01:59:54PM -0800, Katy Huff wrote:
> This agreement was agreed upon by the interim steering committee
> because explicit dedication is superior to implicit dedication. It
> sounds like you believe the opposite. Could you elaborate on why you
> would rather it be implicit?

Because it's less work (no need to archive signed documents, or task
someone with collecting them in the first place).  I don't have to go
to city hall and sign that I'm going to follow the rules when I move
to a new town, the rules just automatically apply to me.  It's nice to
have them layed out for easy reading, but that's as far as I think we
need to go.

> The ejection of dud members is currently left up to the discretion
> of the new committee. When they decide to implement a rule, they can
> certainly do so.

Fair enough.  Until then, the honor system seems like a reasonable
policy.  I think the lack-of-need for a signed document is independent
of teeth, but if you have a policy, it's good to have consequences for
violating it.

I'm also not sure about this:

  While continuing to be an instructor for workshops is welcomed, that
  work is independent of the participation requirements and does not
  count toward the requirement of 3 volunteer hours per week.

If you instruct a class, I'm certainly not going to get on your case
for not putting in an *additional* three hours of volunteer time that
week ;).  Or maybe “three hours a week” rolls over?  If so, I think it
would be good to make the commitment “12 hours a month” or something
else that doesn't roll over.  I don't want all the committee members
working like daemons in January and then taking the rest of the year
off ;).

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 02:03:24PM -0800, Katy Huff wrote:
> concerning your worry about the secretary not being required to keep
> and distribute the meeting minutes in addition to making sure they
> are taken.... please see points 2 and 3 of the secretary's
> responsibilities.

Two is about privately circulating the minutes, and three is more
generic than the minutes.  If I was a lazy, lawerly secretary, I could
argue that archiving and distributing the minutes doesn't fall under
three, since they're already covered explicitly by the chair's 4.

> As far as letting your review cook, that sounds great, except I
> would like to make sure this gets pulled.

If it's already been voted on, I don't see why it can't be merged now.
GitHub will still keep copies of these comments after the PR is
merged, and we can refer back to them later.

**Comment by katyhuff** [comment]
I also don't see why it can't be merged now. I asked greg and damien simply
to approve it.
I try not to pull my own requests...

On Fri, Jan 9, 2015 at 2:11 PM, W. Trevor King <notifications@github.com>
wrote:

> On Fri, Jan 09, 2015 at 02:03:24PM -0800, Katy Huff wrote:
> > concerning your worry about the secretary not being required to keep
> > and distribute the meeting minutes in addition to making sure they
> > are taken.... please see points 2 and 3 of the secretary's
> > responsibilities.
>
> Two is about privately circulating the minutes, and three is more
> generic than the minutes. If I was a lazy, lawerly secretary, I could
> argue that archiving and distributing the minutes doesn't fall under
> three, since they're already covered explicitly by the chair's 4.
>
> > As far as letting your review cook, that sounds great, except I
> > would like to make sure this gets pulled.
>
> If it's already been voted on, I don't see why it can't be merged now.
> GitHub will still keep copies of these comments after the PR is
> merged, and we can refer back to them later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/board/pull/5#issuecomment-69407884>.
>



-- 
http://katyhuff.github.com

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 02:17:49PM -0800, Katy Huff wrote:
> I try not to pull my own requests...

I think that's a good policy, and I try and use it myself.  I was just
saying “don't let my review stand in the way of merging this”.

**Comment by katyhuff** [comment]
These comments might be easier if they were in-line?

Anyway, @wking , I appreciate many of your thoughts on this. I had many of them myself when we were hashing through it. 

I'll address a few of the remaining ones:

> Because it's less work (no need to archive signed documents, or task
> someone with collecting them in the first place)." 

I hear you, but, signing a document seemed an effective way to get the committee to internalize the gravity of their position. Signing something also seemed easy enough for the interim steering committee, in relation to the rest of our efforts. If that level of effort is difficult, then the member has been quite mistaken about what level of effort the committee will require.

> If you instruct a class, I'm certainly not going to get on your case
> for not putting in an *additional* three hours of volunteer time that
> week ;).  Or maybe “three hours a week” rolls over?  If so, I think it
> would be good to make the commitment “12 hours a month” or something
> else that doesn't roll over.  I don't want all the committee members
> working like daemons in January and then taking the rest of the year
> off ;).

I see  your complaint. It does not roll over. However, it was also not meant to be do-or-die. It was meant to be a goalpost. I agree it should be re-visited if folks are gaming the system or finding it too restrictive. However, we want weekly effort from the committee members unless they are on vacation or otherwise. On the topic of instruction.. the explanation is a little simpler.. we made the rule that instruction-doesnt-count in order to be explicit about what qualified as committee effort and what didn't. If you teach a lot of workshops, great, but you still have to do committee work. That's the idea. Hope that helps?

> "I think that's a good policy, and I try and use it myself.  I was just saying “don't let my review stand in the way of merging this”. 

Excellent :+1: Thanks @wking !

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 02:27:46PM -0800, Katy Huff wrote:
> These comments might be easier if they were in-line?

But then I'd have to open a browser ;).  When I get some time I'll
work them up as follow-up pull requests for the committee to review
and accept/reject.

> Anyway, @wking , I appreciate many of your thoughts on this. I had
> many of them myself when we were hashing through it.

Is posting those thoughts and discussions somewhere part of your
“radical transparency” [1] (which sounds great, however you plan to
implement it :).  That's harder to do for phone-based conversations,
but Gentoo's council meets on IRC which makes posting logs really easy
[2].

> > Because it's less work (no need to archive signed documents, or
> > task someone with collecting them in the first place)."
> 
> I hear you, but, signing a document seemed an effective way to get
> the committee to internalize the gravity of their position.

Hopefully the election process filters out the barnstormers.  It's not
like we're electing folks who don't already have reasonable public
track records.

> > If you instruct a class, I'm certainly not going to get on your
> > case for not putting in an *additional* three hours of volunteer
> > time that week ;).  Or maybe “three hours a week” rolls over?  If
> > so, I think it would be good to make the commitment “12 hours a
> > month” or something else that doesn't roll over.  I don't want all
> > the committee members working like daemons in January and then
> > taking the rest of the year off ;).
> 
> I see your complaint. It does not roll over. However, it was also
> not meant to be do-or-die. It was meant to be a goalpost.

That makes sense to me, but in that case I don't think the
“instruction hours don't count” caveat is very helpful.  Teaching a
class is a respectable contribution for one week of SWC ;).

> we made the rule that instruction-doesnt-count in order to be
> explicit about what qualified as committee effort and what
> didn't. If you teach a lot of workshops, great, but you still have
> to do committee work. That's the idea. Hope that helps?

If you teach a workshop every day of the week, I'm fine with you doing
no other non-committee work.  I'd rephrase the goal as “committee work
does not count towards your three hours”, since even passive
committees still do committee work.  The activist distinction just
means you do *other* work to help the organization along, outside of
your committee responsibilities.

[1]: https://github.com/swcarpentry/site/pull/727
[2]: http://wiki.gentoo.org/wiki/Project:Council/Meeting_logs

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 02:39:06PM -0800, W. Trevor King wrote:
> If you teach a workshop every day of the week, …

Oops, I meant “every week of the year”.

**Comment by katyhuff** [comment]
>
>
> Is posting those thoughts and discussions somewhere part of your
> “radical transparency” [1] (which sounds great, however you plan to
> implement it :). That's harder to do for phone-based conversations,
> but Gentoo's council meets on IRC which makes posting logs really easy
> [2].
>

Yes.
We use etherpad.
The chat and discussion leading up to the decisions should be better
recorded than it has been in the past.
I'd like to see that happen. Also, documents like this should be open for
public consumption before we decide on them.. for example.


> > > If you instruct a class, I'm certainly not going to get on your
> > > case for not putting in an *additional* three hours of volunteer
> > > time that week ;). Or maybe “three hours a week” rolls over? If
> > > so, I think it would be good to make the commitment “12 hours a
> > > month” or something else that doesn't roll over. I don't want all
> > > the committee members working like daemons in January and then
> > > taking the rest of the year off ;).
> >
> > I see your complaint. It does not roll over. However, it was also
> > not meant to be do-or-die. It was meant to be a goalpost.
>
> That makes sense to me, but in that case I don't think the
> “instruction hours don't count” caveat is very helpful. Teaching a
> class is a respectable contribution for one week of SWC ;).
>
>
This is definitely true. However, it's not a sufficient contribution for
the SCF Steering Committee, which is primarily tasked with goals that are
not supported by teaching workshops. On this point, the steering committee
was adamant. To help understand the need, universalize the maxim... what if
everyone were spending a lot of time teaching and therefore ignoring their
SCF steering committee responsibilities numerous weeks a year? This
wouldn't help budgets get written, fundraising to get done, instructor
training, mentoring.... you get the idea.

> we made the rule that instruction-doesnt-count in order to be
> > explicit about what qualified as committee effort and what
> > didn't. If you teach a lot of workshops, great, but you still have
> > to do committee work. That's the idea. Hope that helps?
>
> If you teach a workshop every day of the week, I'm fine with you doing
> no other non-committee work. I'd rephrase the goal as “committee work
> does not count towards your three hours”, since even passive
> committees still do committee work.


I don't understand. We want the opposite from what you've just said.
Committee work *is* the three hours.
Instruction doesn't count. Other things directly futhering the goals of the
committee ("committee work") do count.


> The activist distinction just
> means you do *other* work to help the organization along, outside of
> your committee responsibilities.
>
> [1]: https://github.com/swcarpentry/site/pull/727
> [2]: http://wiki.gentoo.org/wiki/Project:Council/Meeting_logs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/board/pull/5#issuecomment-69412275>.
>



-- 
http://katyhuff.github.com

**Comment by gvwilson** [comment]
Looks good to me.

**Comment by katyhuff** [comment]
Thanks @gvwilson . Thanks for the discussion @wking ! For anything that continues to nag at you, PRs are more than welcome on this, but will have to be approved by the steering committee... so...  while the interim steering committee may not have time to get to it before the transition, the actual steering committee may be the best audience for such a PR anyway. Thanks!!

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 03:42:23PM -0800, Katy Huff wrote:
> The chat and discussion leading up to the decisions should be better
> recorded than it has been in the past.  I'd like to see that
> happen. Also, documents like this should be open for public
> consumption before we decide on them.. for example.

Sounds great :).

> > I'd rephrase the goal as “committee work does not count towards
> > your three hours”, since even passive committees still do
> > committee work.
>
> I don't understand. We want the opposite from what you've just said.
> Committee work *is* the three hours.  Instruction doesn't
> count. Other things directly futhering the goals of the committee
> ("committee work") do count.

No, the committee work is covered by the responsibilities and
dedication rules listed earlier.  I'm already assuming folks are
taking care of those (or we're running petions to call for new
elections, or whatever).  The weekly three hours of *additional* time
is what makes our board activist [1].  If you have three hours of
committee time a week, and do no other SWC work, I think you're a
passive member.  The committee is supposed to steer SWC (by being a
functioning committee) and be active (by doing something else besides
being on the committee).

[1]: http://software-carpentry.org/blog/2014/12/plans-for-2015-mentorship-and-assessment.html

**Comment by gvwilson** [comment]
Hi Trevor,
We're looking for 3 hours/week total from steering committee members, not 3 hours on top of committee duties. (Katy is on the steering committee right now, and authored the roles & responsibilities doc, so her interpretation can be trusted.)
Thanks,
Greg

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 04:46:31PM -0800, Greg Wilson wrote:
> We're looking for 3 hours/week total from steering committee
> members, not 3 hours on top of committee duties.

So your distinction between “activist” and “passive” committees is
just the time it takes to discharge committee duties?  Most of the
activities listed in “Methods of Participation” look management-esque,
although “supporting initiatives prioritized by…” sounds pretty open
ended.  If blog posts count, do review comments on pull requests
count?  Maintaining lesson repositories?  I'm not sure where the line
is drawn (maybe it's all SWC work except teaching?  But why single out
teaching?).

Ah, there's also a typo “et.” → “etc.”.

Looking at the general committee-member responsibities, it seems like
there isn't much explicit there (although each of those could take as
much time as you had to throw at it), so I'd count all of that toward
the three hours.  How about:

* Discussions with committee members about committee activity
  *doesn't* count towards your time.

All other SWC work counts towards your time.

Maybe interim committee members can weigh in with estimates of how
much time they're donating now and break it down by:

* committee discussions,
* not a committe discusion, but something for my particular committee
  role,
* and other.

I'd expect your 11 hours falls under the first two, and everything
else you have in your November week falls under “other” [1].  I'd also
expect that 11 hours on purely-committee matters will be unusually
high, once the fact that we have a committee becomes less new.

> Katy is on the steering committee right now, and authored the roles
> & responsibilities doc, so her interpretation can be trusted.

It's not about trust, it's about clarity.  I'm currently having
trouble matching “we're activist” to “no non-committee duties are
required”.

[1]: http://software-carpentry.org/blog/2014/11/where-the-time-goes.html

**Comment by wking** [comment]
On Fri, Jan 09, 2015 at 05:15:01PM -0800, W. Trevor King wrote:
> On Fri, Jan 09, 2015 at 04:46:31PM -0800, Greg Wilson wrote:
> > We're looking for 3 hours/week total from steering committee
> > members, not 3 hours on top of committee duties.
> 
> So your distinction between “activist” and “passive” committees is
> just the time it takes to discharge committee duties?…

I've revised my position to support the “total” in Greg's comment.
See #13.
